### PR TITLE
MAT-4207 add TGT to local storage

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-4207](https://jira.cms.gov/browse/MAT-4207)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [x] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/MainNavBar/MainNavBar.tsx [hunk 1]
```diff
@@ -81,11 +107,15 @@ const MainNavBar = () => {
                 </ListItem>
                 <ListItem className="activity-button">
                   <button
-                    onClick={() => setDOpen(true)}
+                    onClick={() =>
+                      setDOpen(!tgtValueFromStorage ? true : false)
+                    }
                     data-testid="UMLS-connect-button"
                   >
                     <div className={TGT ? "active" : "inactive"} />
```
Comment (mcmcphillips) [inline src/components/MainNavBar/MainNavBar.tsx]: ClassName check should also check for local storage like the next line in order to apply the green bubble for styling

**Review by mcmcphillips — CHANGES_REQUESTED** [review]

File: src/components/MainNavBar/MainNavBar.tsx (hunk 1 above)
Comment (sb-cecilialiu) [reply src/components/MainNavBar/MainNavBar.tsx]: @mcmcphillips Thank you so much Matt! That's a good catch! I just pushed in the change.

File: src/components/MainNavBar/MainNavBar.test.tsx [hunk 2]
```diff
@@ -226,4 +240,52 @@ describe("UMLS Connection Dialog", () => {
       });
     });
   });
+
+  test("Should not render ULMSDialog when user has valid TGT", async () => {
```
Comment (jkotanchik-SB) [inline src/components/MainNavBar/MainNavBar.test.tsx]:
```suggestion
  test("Should not render UMLSDialog when user has valid TGT", async () => {
```
`ULMS` -> `UMLS`

File: src/components/MainNavBar/MainNavBar.tsx [hunk 3]
```diff
@@ -16,6 +16,7 @@ import UserProfile from "./UserProfile";
 import UserAvatar from "./UserAvatar";
 import UMLSDialog from "./ULMSDialog";
```
Comment (jkotanchik-SB) [inline src/components/MainNavBar/MainNavBar.tsx]: Do you mind renaming this component to `UMLSDialog`? If it causes too many issues, we can write it up as a tech debt story.

Comment (sb-cecilialiu) [reply src/components/MainNavBar/MainNavBar.tsx]: @jkotanchik-SB Sure just did. Thanks so much Joe, it's a name hard to remember

**Review by jkotanchik-SB — APPROVED** [review]

**Review by mcmcphillips — APPROVED** [review]

**Review by adongare — APPROVED** [review]
